server.js

```diff
@@ -4,6 +4,8 @@ const WebpackDevServer = require('webpack-dev-server');
 const config = require('./webpack.config');
 const { oneLine, stripIndent } = require('common-tags');
 const port = process.env.PERFHTML_PORT || 4242;
+const host =
+  process.env.PERFHTML_PUBLICHOST === 'true' ? '0.0.0.0' : 'localhost';
```

Comment:
If `PERFHTML_PUBLICHOST` contains the name of the real public host, we can do:
```js
const host = process.env.PERFHTML_PUBLICHOST || 'localhost';
```
then...

Reply:
Hm, interesting proposal! Thanks. 😄 It does seem cleaner from the point of view of `WebpackDevServer`, however from a developer perspective it would imply doing something hacky like:

```bash
PERFHTML_PUBLICHOST=`echo $GITPOD_WORKSPACE_URL | sed 's_https://_https://4242-_'` yarn start
```